operations/iscsi_sbps/Node_Personalization_Verification.md

```diff
@@ -1,242 +1,13 @@
-      },
-      "configuration": {
-        "limit": "",
-        "name": "iscsi-sbps-targets-config"
-      },
-      "debug_on_failure": false,
-      "logs": "ara.cmn.surtur.hpc.amslabs.hpecorp.net/?label=iscsi-sbps-targets-config",
-      "name": "iscsi-config-20240831210845",
-      "status": {
-        "artifacts": [],
-        "session": {
-          "completion_time": null,
-          "ims_job": null,
-          "job": null,
-          "start_time": "2024-08-31T21:08:47",
-          "status": "pending",
-          "succeeded": "none"
-        }
-      },
-      "tags": {},
-      "target": {
-        "definition": "dynamic",
-        "groups": [],
-        "image_map": []
-      }
-    }
-    ```
-
-Note:
-For specific target worker nodes selection for node personalization (instead of default: to all workers)
-use CFS option `--ansible-limit` with xnames of the worker nodes while creating a session.
-
-```bash
-cray cfs v3 sessions create --name <session_name> --configuration-name <config_name> --ansible-limit <xname1,xname2,...>
-```
-
-### Monitor CFS session
-
-#### Check status of CFS session
-
-See [View CFS Sessions](CFS_Sessions.md) for more information.
-
-```bash
-cray cfs v3 sessions describe "${SESSION}" --format toml
-```
-
-```toml
-debug_on_failure = false
-logs = "ara.cmn.surtur.hpc.amslabs.hpecorp.net/?label=iscsi-sbps-targets-config"
-name = "iscsi-config-20240831210845"
-
-[ansible]
-config = "cfs-default-ansible-cfg"
-limit = ""
-passthrough = ""
-verbosity = 0
-
-[configuration]
-limit = ""
-name = "iscsi-sbps-targets-config"
-
-[status]
-artifacts = []
-
-[tags]
-
-[target]
-definition = "dynamic"
-groups = []
-image_map = []
-
-[status.session]
-completion_time = "2024-08-31T21:09:19"
-job = "cfs-b840455c-e919-4656-b64a-44d433f082dc"
-start_time = "2024-08-31T21:08:47"
-status = "complete"
-succeeded = "True"
-```
-
-#### Track the status of Ansible playbooks
-
-Make sure that all the CFS play books have completed successfully.
-Look for the message "All playbooks completed successfully" in the CFS Ansible container log.
-
-See [Troubleshooting](Track_the_Status_of_a_Session.md) for more information.
-
-1. Get name of CFS pod.
-
-    ```bash
-    CFSPOD=$(kubectl get pods --no-headers -o custom-columns=":metadata.name" -n services -l cfsession=${SESSION}) ; echo $CFSPOD
-    ```
-
-    Example output:
-
-    ```text
-    cfs-b840455c-e919-4656-b64a-44d433f082dc-5d4jp
-    ```
-
-1. View the last line of Ansible container log.
-
-    ```bash
-    kubectl logs -n services "${CFSPOD}" -c ansible --tail=1
-    ```
-
-    Example of output if CFS was successful:
-
-    ```text
-    All playbooks completed successfully
-    ```
-
-### Post-personalization verification
-
-(`ncn-w#`) Run these checks on each worker node, in order to verify that it was configured correctly.
-
-1. Verify that the SBPS Marshal Agent is running without any errors.
+## Verify that the SBPS Marshal Agent is running without any errors.
 
```

Comment:
What specifically are they looking for in the output? What needs to be present for this to be looking correct? There is example output give, but it's not clear which parts of it need to be verified. I see that it says up above to check that it's running, which I presume means under the Active field, but that should be stated explicitly. As for "without errors", where are they supposed to verify that? Does that mean it is running rather than failed? Or does it mean an error message in the systemctl status log entries shown in the output (below the part that is shown in the example)?

Reply:
Marking this resolved because the work on the verification page is moving to [CASMPET-7640](https://jira-pro.it.hpe.com:8443/browse/CASMPET-7640). Once a PR is open for that, I'll add comments for the relevant issues identified in this PR.